Documentation/ceph-object.md

```diff
@@ -106,6 +106,46 @@ To directly retrieve the secrets:
 kubectl -n rook-ceph get secret rook-ceph-object-user-my-store-my-user -o yaml | grep AccessKey | awk '{print $2}' | base64 --decode
 kubectl -n rook-ceph get secret rook-ceph-object-user-my-store-my-user -o yaml | grep SecretKey | awk '{print $2}' | base64 --decode
 ```
+## Create an Object Bucket Claim
+
+Creating an Object Bucket Claim (OBC) causes the Rook-Ceph bucket provisioner to create a new bucket or to grant access to an existing bucket. OBCs reference a Storage Class which is created by a cluster administrator. The Storage Class defines the object storage system, the bucket retention policy, and, when granting access to existing (brownfield) buckets, the name of the bucket. If a new bucket is desired then the OBC contains the bucket name or a prefix used for a random bucket name. See the [Object Bucket Claim Documentation](ceph-object-bucket-claim.md) for more detail on the `CephObjectBucketClaims`.
```

Comment:
This may need to change entire flow of the entire doc, even for accessing the object store part as well. Can we take altogether as different patch

Reply:
I think this is critical to get the new flow clear. Instead of trying to explain this in detail, I made a pass at these changes and the commit is pushed [here](https://github.com/travisn/rook/commit/ba3d9a8df41b8a0e90fd7961f04868933f8619a1). Want to cherry pick this commit and include it in the PR?